### DISCUSSION TITLE
How can we adjust the word count for the excerpt on the homepage?

### DISCUSSION THREAD

**Discussion by bershatsky** [adityatelange/hugo-PaperMod#604] [Q&A]
I thought the read more tag in the post would take care of that, but it doesn't.

Thoughts?

**Comment by chrisrummel** [comment]
According to [summary-documentation](https://gohugo.io/content-management/summaries/) it should be set by `summaryLength` with a default of "70". Setting it to "200" doesn't have any effect, the maximum summary I get is two lines of text. Try reducing the browser window size and the word count will decrease while staying at two lines, no matter to which value you set `summaryLength` to.

**Comment by danielfdickinson** [comment]
https://github.com/adityatelange/hugo-PaperMod/blob/42a23c96e26fa6b8d9eee0515e318c383e736bd1/assets/css/common/post-entry.css#L14

Is the issue. It clamps the box size to three lines (which because of font size gives you two lines of text).

**Reply by bershatsky** [reply]
I found this line, adjusted the 3 to another larger number - no change. What am I missing?

**Reply by danielfdickinson** [reply]
If you are using the ``exampleSite`` content, it also has

https://github.com/adityatelange/hugo-PaperMod/blob/5aa358cab1eea52bdf7c97972fbccd44cdeb271a/content/posts/markdown-syntax.md?plain=1#L16

(note the ``<!--more-->``. That limits the summary to that point, regardless of the ``summaryLength`` setting or the line-clamp.

If I remove the ``<!--more-->`` and increase the line clamp (note that there are three (3) occurrences), I see:

![image](https://user-images.githubusercontent.com/20735818/139017802-a060398b-e188-42ba-a221-9a9dcbc811cd.png)

on the second page of the exampleSite.

For the first article (``Installation | Update``) there is a ``summary: `` frontmatter defined, which is another way of setting the summary. See:

https://github.com/adityatelange/hugo-PaperMod/blob/5aa358cab1eea52bdf7c97972fbccd44cdeb271a/content/posts/papermod/papermod-installation.md?plain=1#L3

As you can see there are several separate ways the summary has been set for ``exampleSite`.

**Reply by danielfdickinson** [reply]
> I found this line, adjusted the 3 to another larger number - no change. What am I missing?

Do note there are three separate lines with ``-webkit-line-clamp`` set, in case that is buried in the above reply.

**Reply by chrisrummel** [reply]
> Do note there are three separate lines with `-webkit-line-clamp` set, in case that is buried in the above reply.

Actually there are 4, and I guess the fourth is responsible:

```
grep -n webkit-line-clamp themes/papermod/assets/css/common/post-entry.css 
14:    -webkit-line-clamp: 3;
25:    -webkit-line-clamp: 3;
33:    -webkit-line-clamp: unset;
67:    -webkit-line-clamp: 2;
```
